package.json

```diff
@@ -78,26 +78,23 @@
     "cross-fetch": "^3.0.4",
     "date-fns": "^2.24.0",
     "isomorphic-fetch": "^3.0.0",
+    "lodash": "^4.17.21",
     "lodash.map": "^4.6.0",
-    "lodash.snakecase": "^4.1.1"
+    "lodash.snakecase": "^4.1.1",
```

Comment:
If you have `lodash`, aren't `lodash.map` and `lodash.snakecase` redundant?

Reply:
I removed redundant imports, it seems to work fine